docs/content/getting-started.mdx

```diff
@@ -84,15 +93,15 @@ There are a few ways to materialize an asset:
    Serving dagit on http://127.0.0.1:3000 in process 70635
    ```
 
-3. Navigate to <http://localhost:3000> in your web browser to view your asset.
+3. Navigate to <http://localhost:3000> in your web browser to view your assets.
 
-4. Click the **Materialize All** button to launch a run that materializes the asset:
+4. Click the **Materialize All** button to launch a run that materializes the assets:
 
    <img src="/images/getting-started/materialize-asset-in-dagit.gif" />
 
 ### Using the Dagster Python API
 
-You can also use the [Dagster Python API](/concepts/ops-jobs-graphs/job-execution#python-apis) materialize the asset as a script.
+You can also use the [Dagster Python API](/concepts/ops-jobs-graphs/job-execution#python-apis) materialize the assets as a script.
```

Comment:
It looks like the Python example below only references `cereals`, not `nabisco_cereals`.